.gitattributes

```diff
@@ -1,6 +1,13 @@
+# Default behaviour, for if core.autocrlf isn't set
+* text=auto
+
+# Binary files
 boot/ocamlc binary
 boot/ocamllex binary
 boot/ocamldep binary
+*.gif
+*.png
+*.tfm
```

Comment:
I notice, however, that I've been a twit - I've listed the wildcards and omitted the `binary`! A technical error, though - they will be defaulting to binary anyway (they were anyway, even without `* text=auto`). I was doing some tinkering with attributes for `tools/check-typo` to skip files which git knows are binary, and I'll fix these three lines there...